orbs/shared/jobs/e2e.yaml

```diff
@@ -0,0 +1,23 @@
+description: Run E2E tests in a Kubernetes cluster
+parameters:
+  vault_address:
+    description: Vault Instance to use
+    type: string
+    default: ""
```

Comment:
is the reason we're not providing a default because of oss?

Reply:
Yep

Reply:
This will be passed in by stencil via box config anyways so default will rarely ever be hit